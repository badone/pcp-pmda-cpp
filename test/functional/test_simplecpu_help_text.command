TEMP_DIR=`mktemp -d -t test_simplecpu_help_text-XXXXXXXXXX`
./pmdasimplecpu --export-all "$TEMP_DIR"
if [ $? -ne 0 ]; then exit; fi
dbpmda -e -n "$TEMP_DIR/root" <<EOF
open pipe pmdasimplecpu
text simplecpu.ticks.total
text simplecpu.ticks.cpu0
close
EOF
rm -rf "$TEMP_DIR"
