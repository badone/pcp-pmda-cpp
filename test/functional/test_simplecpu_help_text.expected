open pipe pmdasimplecpu
Start pmdasimplecpu PMDA: ./pmdasimplecpu
text simplecpu.ticks.total
PMID: 129.0.0
[The amount of time spent in various states]
The amount of time spent in various states
text simplecpu.ticks.cpu0
PMID: 129.0.1
[The amount of time spent in various states]
The amount of time spent in various states
close
